src/replication/replicator.rst

```diff
@@ -21,6 +21,11 @@ Replicator Database
    anymore. There are new replication job states and new API endpoints
    ``_scheduler/jobs`` and ``_scheduler/docs``.
 
+.. versionchanged:: 3.2.0 Fair share scheduling was
+   introduced. Multiple ``_replicator`` databases get an equal chance
```

Comment:
Is the line wrap important here? Should `introduced` go to the line above?

Reply:
You're right. It should go in the line above. Updated.